docs/eog/case-studies-cw20-infrastructure.rst

```diff
@@ -0,0 +1,67 @@
+.. _CW20-Infrastructure: 
+
+CW20 Infrastructure
+====================
+
+We had already organised registration for the in-person event through `Eventbrite <https://www.eventbrite.com/>`_. 
+We sent emails to registrants through Eventbrite to notify them of our decision making every step of the way. 
+Because of the unconference nature of the event, we even asked for participant input to our contingency investigation. 
+As we worked to reformat the event and move it online, we gave registrants time to cancel and collect full refunds. 
+We also reduced the cost of the event (from £250 to £50) and issued partial refunds through Eventbrite to participants who wanted to stay and participate in the virtual event. 
+We took the partial refund route in order to maintain continuity with our registrants, but another option could have been to outright cancel the Eventbrite event, issue full refunds and start from scratch. 
```

Comment:
Might be worth saying - why the other option might have been worth considering - I think there was a different fee impact if I remember.